### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by reinzor**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,14 +1,12 @@
-sudo: required
 language: generic
-dist: trusty
+services:
+  - docker
 
-matrix:
-  include:
-  - name: "Kinetic"
```
Comment (LoyVanBeek) [inline .travis.yml]: No more Kinetic? HERO runs kinetic AFAIK

Comment (reinzor) [reply .travis.yml]: Damn ur fast, I was just checking whether this works under melodic (18.04). Since pip apparently seems to have problems on 16.04.

Comment (reinzor) [reply .travis.yml]: also not working :(

Comment (LoyVanBeek) [reply .travis.yml]: > Damn ur fast

Tuesday evening...

**Pull request closed without merging**